tensorflow_addons/image/filters.py

```diff
@@ -109,3 +109,90 @@ def func2():
         y = tf.cast(y, tf.int32)
 
         return y
+
+
+@tf.function
+def mean_filter2d(image, filter_shape=(3, 3)):
+    """This method performs Mean Filtering on image. Filter shape can be user
+    given.
+
+    This method takes both kind of images where pixel values lie between 0 to
+    255 and where it lies between 0.0 and 1.0
+    Args:
+        image: A 3D `Tensor` of type `float32` or 'int32' or 'float64' or
+               'int64 and of shape`[rows, columns, channels]`
+
+        filter_shape: Optional Argument. A tuple of 2 integers (R,C).
+               R is the first value is the number of rows in the filter and
+               C is the second value in the filter is the number of columns
+               in the filter. This creates a filter of shape (R,C) or RxC
+               filter. Default value = (3,3)
+
+     Returns:
+         A 3D mean filtered image tensor of shape [rows,columns,channels] and
+         type 'int32'. Pixel value of returned tensor ranges between 0 to 255
+    """
+
+    def _normalize(li):
```

Comment:
Can we pull this out as two functions now use the same normalize func? Probably should keep `_` prefix and its `tf.function` decorator